region/brinstar/blue.json

```diff
@@ -1494,6 +1858,103 @@
               ]
             }
           ]
+        },
+        {
+          "from": 5,
+          "to": [
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    "h_canArtificialMorphIBJ",
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "SpaceJump",
+                      "canOffScreenMovement"
+                    ]}
+                  ],
+                  "note": "Overload PLMs by repeatidly bombing the Power Bomb blocks or the side of the crumble block, then enter through the crumble block.",
+                  "devNote": "The off screen movement is just during an IBJ, and only if wall jumps are disabled. But the IBJ is long and Samus is covered for most of it."
+                }
+              ]  
+            },
+            {
+              "id": 4,
+              "strats": [
+                {
+                  "name": "G-Mode Morph",
+                  "notable": false,
+                  "requires": []
+                },
+                {
+                  "name": "G-Mode Morph with Power Bomb",
+                  "notable": false,
+                  "requires": [ {"ammo": {"type": "PowerBomb","count": 1}} ],
+                  "clearsObstacles": ["B","C"],
+                  "note": "The blocks will not break if PLMs are already overloaded. Exiting G-Mode before the Power Bomb explodes will ensure they break."
+                }
+              ]  
+            }
+          ]
+        },
+        {
+          "from": 6,
+          "to": [
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "G-Mode Morph Power Bomb the Item",
+                  "notable": false,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 4},
+                    "canRiskPermanentLossOfAccess",
+                    {"ammo": {"type": "PowerBomb","count": 1}},
+                    "h_canArtificialMorphMovement",
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "SpaceJump"
+                    ]}
+                  ],
+                  "note": [
+                    "Overload PLMs with a single Power Bomb by hitting the item at the right end of the room.",
+                    "This can be done with a single precisely placed Power Bomb. There should be 4 empty tiles between Samus and the left of the blocks when the bomb is placed."
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph Touch the Item",
+                  "notable": false,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 4},
```

Comment:
I don't think this is modelled correctly still.
1) Logic wouldn't know the player has the item and can be at the top door.
2) Logic doesn't know that this is a one time use strat.  

It needs some kind of `{remoteItemCollection: 4}` object or something.

Reply:
I could add an obstacle to enable it to grab the item on the way, but there's no way to force the item collection. I was discussing a remote collection with Maddo and he had major concerns with it. I'll ask him to respond on that.

I wonder if we could have a strat to or from a node that forces item pickup at that node (like going through Botwoon E-Tank tunnel cant skip the item). We could also add a way to indicate that a strat can only be done once (like the tube gravity jump)

Reply:
We wouldn't have to force item collection if we just be sure that the strat can only be used to collect the item and nothing else. That could be done by having the strat go from the junction (6) to the item node (4) while clearing a new abstract obstacle; then there can be a strat from 4 back to 1 just requiring that obstacle.

I don't think I would try to model strats using the item to pass through the room to the other door (while collecting it). We could consider that later but it would seem to require some schema changes and not be usable in randomizer logic anyway due to the risk of permanent loss of access.

Reply:
Currently I've been putting in all strats that I can think of. Those that go to the item and back are listed without `canRiskPermanentLossOfAccess`, and those that do require it.

Reply:
We could add a new strat-level property "itemsCollected" (similar to "obstaclesCleared"), to represent items that are (and must be) collected as a side-effect of performing the strat. I don't think Map Rando would be able to consume this data in the foreseeable future, but it could be useful anyway to help make the model more complete, since it could be useful later in some way that we haven't thought of